### PR TITLE
Improve documentation

### LINKED ISSUE #13: Document how to use the bots and what can and cannot be expected
Provide some simple documentation that explains how to use the bots, what their features/capabilities are.

In no particular order, here are some random things I learned when trying to use them today:
- there are actually 2 different bots.  One of the is only for _all of the AgileVentures Slack instance_ and the other runs _1 per channel_
- using a bot means it's tied up and no one can use it to vote on anything else until you're done with it

How do you "run" each bot?
How do you "release" each bot?
How do you get the results?
What if you have trouble?  Can you kill/restart it?
Can you 'undo' or 'subtract' a vote or do you have to just re-do the whole vote?

What's the syntax?   Is there a help command or flag?

### PR DESCRIPTION
**Pull request by arreche**
This pull request gives more details about how to use the Slack app and set up a development environment.

Fix #13

### CODE REVIEW

**Comment by arreche** [comment]
@arun1595 Please take a look to this!

**Comment by arreche** [comment]
@weedySeaDragon Please take a look to this!

File: Readme.md [hunk 1]
```diff
@@ -1,26 +1,47 @@
 Slack app for Voting on Stories and Tickets remotely and asynchronously.
 
+## Usage
+
+1. Install this app to your Slack team visiting http://asyncvoter.tk and pressing on the `Add to Slack` button.
```
Comment (arun1595) [inline Readme.md]: @arreche Can we explicitly mention in a step that the contributor has to create a new slack team? 'To your slack team' isn't too clear IMO

File: Readme.md [hunk 2]
```diff
@@ -1,26 +1,47 @@
...
+2. Run a voting session on an issue launching the new `/vote issue` command in any channel.
+3. Submit your vote pressing on any of the three difficulty buttons (Easy, Medium or Hard).
+4. Finish and reveal the result of the voting session by pressing on the `Reveal` button. (We recommend to wait until you will receive 3 or more votes)
+
+## Known Issues
+
+- Votes cannot be reseted. But users can vote multiple times on the same issue.
```
Comment (arun1595) [inline Readme.md]: Votes cannot be _reset_

File: Readme.md [hunk 3]
```diff
@@ -1,26 +1,47 @@
...
+- It supports parallel voting sessions in different channels only.
+
+## Questions/feedback?
+
+Create an issue in this repo, or contact https://agileventures.slack.com/messages/async_voter
```
Comment (arun1595) [inline Readme.md]: or contact us at https://agileventures.slack.com/messages/async_voter

File: Readme.md [hunk 4]
```diff
@@ -1,26 +1,47 @@
...
- - Create an Slack team
- - Place tokens in `.env` file
- - Enable `Interactive Messages` and `Slash Commands`
+2. Run tests `npm test`
 
-3. Install dependencies `npm install`
+3. Run the server `npm run dev`
 
-4. Run tests `npm test`
+You **need** to set up your own Slack **team** and Slack **app** to continue!
```
Comment (arun1595) [inline Readme.md]: Ah! here we have it :)

File: Readme.md (hunk 1 above)
Comment (weedySeaDragon) [inline Readme.md]: Isn't it possible to add this to an existing Slack team?
(How did it get added to AgileVentures?)

Comment (arun1595) [inline Readme.md]: @weedySeaDragon Only slack team admins can add apps to channels. That's why a contributor needs to create his/her own team.

Comment (arun1595) [inline Readme.md]: @weedySeaDragon Only a slack team admins can add apps to channels. That's why a contributor needs to create his/her own team.

Comment (arreche) [inline Readme.md]: This document is for both `end users` and `contributors/developers`

Comment (arreche) [inline Readme.md]: @arun1595 As a developer/contributor Is up to you creating a new or using an existing team for testing purposes.

Comment (arreche) [inline Readme.md]: @weedySeaDragon Absolutely. End users would like to add/install this app to their teams.

**Comment by arreche** [comment]
@arun1595 and @weedySeaDragon. Thanks so much for your feedback!

I am going to update this pull request with your valuable comments :)

**Comment by weedySeaDragon** [comment]
I vote for accepting this PR and merging it in so that the README is updated.   

Additional improvements to it can be done with additional PRs.  

Nice work, @arreche !

**Pull request merged**